prov/udp/src/udpx_init.c

```diff
@@ -88,7 +100,16 @@ static void udpx_getinfo_ifs(struct fi_info **info)
 	}
 	freeifaddrs(ifaddrs);
 
-	if (head) {
+	if (head || loopback) {
+		if(!head) { /* loopback interface only? */
+			head = loopback;
+		} else {
+			/* append loopback interfaces to tail */
+			assert(tail);
+			assert(!tail->next);
+			tail->next = loopback;
+		}
```

Comment:
here is implemented 2 lists & loopback info's are attached to end of "regular" list